config/core/build-configs.yaml

```diff
@@ -706,11 +706,6 @@ build_configs:
     <<: *kselftest-tree
     branch: 'next'
 
-  lee_android_3.18:
-    tree: lee
```

Comment:
This branch was the only one using the `lee` tree, so we might also drop it.

@lag-linaro Are there other branches in your tree you would like to have covered by KernelCI?

Reply:
No, not right now, thank you.

Reply:
I will update PR accordingly.